cmd/server-main.go

```diff
@@ -231,7 +231,7 @@ func initSafeMode(ctx context.Context, newObject ObjectLayer) (err error) {
 	}
 
 	// allocate dynamic timeout once before the loop
-	configLockTimeout := newDynamicTimeout(3*time.Second, 5*time.Second)
```

Comment:
We can keep this same as iamLockTimeout @klauspost 1second is too low and makes it very chatty when the server is coming up. Let's keep it similar to IAM lock timeout